appcenter-crashes/ios/appcenter-crashes.podspec

```diff
@@ -10,10 +10,10 @@ Pod::Spec.new do |s|
   s.homepage          = package['homepage']
   s.documentation_url = "https://docs.microsoft.com/en-us/appcenter/"
 
-  s.author           = { 'Microsoft' => 'appcentersdk@microsoft.com' }
+  s.author            = { 'Microsoft' => 'appcentersdk@microsoft.com' }
 
-  s.source = { :git => "https://github.com/Microsoft/AppCenter-SDK-React-Native.git" }
-  s.source_files = "appcenter-crashes/ios/AppCenterReactNativeCrashes/*.{h,m}"
+  s.source            = { :git => "https://github.com/Microsoft/AppCenter-SDK-React-Native.git" }
+  s.source_files      = "AppCenterReactNativeCrashes/**/*.{h,m}"
```

Comment:
This seems to work for cocopods project too. However before I merge, can you elaborate why it works to remove the beginning of the path, and what about the stars?

Reply:
Yep: the source_files path is related to the root of the .podspec file. So we want to include in the podspec all .h and .m files (including sub-folders thus **/*.{h.m}) in AppCenterReactNativeCrashes folder starting from the root. 

Starting from 2 levels up in the filesystem path would break the spec since source files would not be found.